.pipelines/templates/.builder-release-template.yaml

```diff
@@ -113,6 +113,11 @@ steps:
   - bash: |
         OS_DISK_URI="$(cat packer-output | grep "OSDiskUri:" | cut -d " " -f 2)" && \
         MANAGED_SIG_ID="$(cat packer-output | grep "ManagedImageSharedImageGalleryId:" | cut -d " " -f 2)" && \
+        PKR_RG_NAME="$(cat packer-output | grep "ResourceGroupName" | cut -d "'" -f 2 | head -1)" && \
+        captured_sig_version="$(cat vhdbuilder/packer/settings.json | grep "captured_sig_version" | awk -F':' '{print $2}' | awk -F'"' '{print $2}')" && \
```

Comment:
why not in CPAS ? like the rest ?

Reply:
@djsly I am not sure what a CPA is. Do you mean in all capital letters? If so, this variable is this way all throughout the codebase, I have wondered that myself.

Reply:
CAPS

Reply:
Yes I am not sure why it is that way, I was just trying to keep it consistent with the rest of the repo.

Reply:
we can align towards CAPS here